src/main/java/seedu/duke/task/Task.java

```diff
@@ -19,6 +19,15 @@ public boolean isDone() {
     protected boolean isDone;
     protected LocalDate date;
     protected TeamMember member;
+    protected String priority;
+
+    public String getPriority() {
+        return priority;
+    }
+
```

Comment:
Dude over here we passing a string for the priority ryt, You want to change it to an enumeration??

Reply:
Agree with this

Reply:
I thought of it also but I realized it's more of an improvement since as of now there isn't a need for it.
We have no feature planned that will use the enums, (e.g sort by priority).
Also, the error handling is quite tedious to implement if we use enums.
I will implement it in the final run once we're done with a working 2.0.